code/modules/reagents/newchem/food.dm

```diff
@@ -142,4 +142,20 @@ datum/reagent/mugwort/on_mob_life(var/mob/living/M as mob)
 			M.adjustBruteLoss(-1*REM)
 			M.adjustFireLoss(-1*REM)
 	..()
+	return
+
+/datum/reagent/porktonium
+	name = "Porktonium"
+	id = "porktonium"
+	description = "A highly-radioactive pork byproduct first discovered in hotdogs."
+	reagent_state = LIQUID
+	color = "#AB5D5D"
+	metabolization_rate = 0.2
+
+datum/reagent/porktonium/overdose_process(var/mob/living/M as mob)
+	if(volume > 125)
```

Comment:
You need to set overdose_threshold on the reagent define to trigger the overdose processing procs.